File: generate.core/ViewModels/CurrentMigrationStatus.cs
using System;
using generate.core.Models.App;

namespace generate.core.ViewModels.App
{
    public class CurrentMigrationStatus
    {
        public string ODSMigrationStatusCode { get; set; }
        public DateTime? ODSLastMigrationTriggerDate { get; set; }
        public int? ODSLastMigrationDurationInSeconds { get; set; }
        public DateTime? ODSLastMigrationHistoryDate { get; set; }
        public string ODSLastMigrationHistoryMessage { get; set; }

        public string RDSMigrationStatusCode { get; set; }
        public DateTime? RDSLastMigrationTriggerDate { get; set; }
        public int? RDSLastMigrationDurationInSeconds { get; set; }
        public DateTime? RDSLastMigrationHistoryDate { get; set; }
        public string RDSLastMigrationHistoryMessage { get; set; }

        public string ReportMigrationStatusCode { get; set; }
        public DateTime? ReportLastMigrationTriggerDate { get; set; }
        public int? ReportLastMigrationDurationInSeconds { get; set; }
        public DateTime? ReportLastMigrationHistoryDate { get; set; }
        public string ReportLastMigrationHistoryMessage { get; set; }
        public string UserName { get; set; }
    }
}

File: generate.web/ClientApp/src/app/models/app/currentMigrationStatus.ts
export interface CurrentMigrationStatus {

    odsMigrationStatusCode: string;
    odsLastMigrationTriggerDate: Date;
    odsLastMigrationDurationInSeconds: number;
    odsLastMigrationHistoryDate: Date;
    odsLastMigrationHistoryMessage: string;

    rdsMigrationStatusCode: string;
    rdsLastMigrationTriggerDate: Date;
    rdsLastMigrationDurationInSeconds: number;
    rdsLastMigrationHistoryDate: Date;
    rdsLastMigrationHistoryMessage: string;

    reportMigrationStatusCode: string;
    reportLastMigrationTriggerDate: Date;
    reportLastMigrationDurationInSeconds: number;
    reportLastMigrationHistoryDate: Date;
    reportLastMigrationHistoryMessage: string;
    userName: string;
    
}

